addons/website_forum/static/src/js/website_forum.js

```diff
@@ -118,35 +118,71 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             var editorKarma = $textarea.data('karma') || 0; // default value for backward compatibility
             var hasFullEdit = parseInt($("#karma").val()) >= editorKarma;
             var toolbar = [
-                ['style', ['style']],
-                ['font', ['bold', 'italic', 'underline', 'clear']],
-                ['para', ['ul', 'ol', 'paragraph']],
-                ['table', ['table']],
+                [
+                    [
+                        'ParagraphButton',
+                        'Heading1Button',
+                        'Heading2Button',
+                        'Heading3Button',
+                        'Heading4Button',
+                        'Heading5Button',
+                        'Heading6Button',
+                        'PreButton',
+                    ],
+                ],
+                [
+                    'BoldButton',
+                    'ItalicButton',
+                    'UnderlineButton',
+                    'RemoveFormatButton'
+                ],
+                [
+                    'OrderedListButton',
+                    'UnorderedListButton',
+                    [
+                        'AlignLeftButton',
+                        'AlignCenterButton',
+                        'AlignRightButton',
+                        'AlignJustifyButton',
+                    ],
+                ],
+                ['TableButton'],
             ];
             if (hasFullEdit) {
-                toolbar.push(['insert', ['link', 'picture']]);
+                toolbar.push([
+                    'OdooLinkButton',
+                    'UnlinkButton',
+                    'OdooMediaButton',
+                ]);
             }
-            toolbar.push(['history', ['undo', 'redo']]);
-
+            toolbar.push(['UndoButton', 'RedoButton']);
             var options = {
-                height: 200,
-                minHeight: 80,
-                toolbar: toolbar,
-                styleWithSpan: false,
-                wrapperClass: 'note-editable flex-grow-1 ',
+                // toolbarLayout: toolbar,
```

Comment:
Two options: the custom toolbar defined above, or the default one. Depending on what's decided I'll remove one part of the code or the other.

Reply:
I don't understand the question. The toolbar in website forum should be the same on master compared to 14.0. If this matches the default toolbar then fine, but if it doesn't then I think you have your answer.

Reply:
The toolbar changed everywhere else in the meantime... We have a floating toolbar now so it's different in any case. Hence my question.

Reply:
Then the question is for @sbuhl I guess.

Reply:
@Zinston and @dmo-odoo 
May I ask you to be more specific in your question. My Google translate from JS to screenshot with red arrows isn't working.
What are you asking? The layout of the toolbar on the page `/forum/help-1/question/how-to-design-a-nice-forum-toolbar` ?
I'd say the layout of the v13.3 was nice: [https://tinyurl.com/y4esl7rj](V13.3 forum)

Reply:
The question is whether we _need_ to redefine the toolbar for website forum so it's custom with the elements listed above, or if it's ok to simply use the default toolbar like in note.
David was saying we have to have the same as in 14.0 but that's not really true since we now have a floating toolbar that is effectively different in master than in 14.0, no matter the module.
I personally don't understand why we would need a different toolbar for website forum. Maybe Summernote required us to redefine it every time so we had no default?
Anyway it's your call. Please consider that custom toolbar = more code though ;-)

Reply:
Don't know if you got your answer here: for me, the design can change if it is still clear it's an area to post a formatted message of course (the ideal would be to remember the meeting we had last year: if possible, having a toolbar with bootstrap elements for this case so that the editor is integrated and fits the user's theme).

As for the fact it's custom with the listed elements: the logic should be the same. For example, I know the possibility to add a link or an image is only there for users with enough karma.

Reply:
Ok so I'll do that: floating custom toolbar with the same configuration as before and the conditional Link & Media buttons.
I'll work on the CSS too because it's indeed quite unclear right now.